prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
```

Comment:
nit: I'd suggest slightly separating the normative and informative parts of this more sharply, and putting the normative part first. Maybe something like:

```
Each [=prerendering browsing context=] has a <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, which is an [=ordered map=] from [=origin=] to [=client hint set=].

<p class="note">This stores which client hints each origin has opted into receiving, until it can be copied to the global [=Accept-CH cache=] when [=prerendering browsing context/finalize activation|activation is finalized=].</p>
```

Reply:
Although I like Jeremy's rephrasing (modulo preferring plurals for the map declaration), I think it's a bit nicer to keep the explanation as part of the same non-note paragraph. We are debating really fine points of style here, but I tend to prefer notes to be more like "Note: some interesting supplementary information" instead of "Note: some basic information".

Reply:
This part is describing the usage of prerender-scoped CH cache, based on https://www.rfc-editor.org/rfc/rfc8942(This should be a specification).
I'd like this to be a part of the "normative paragraph", and wonder how I can make it more normative..